data/spservicing.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Select Portfolio Servicing, inc. login",
```

Comment:
Nit: `inc.` should be `Inc.`. Also can you get rid of "login"? `name` is supposed to be just the name of the website, not the name of the page or a description. (This isn't your fault, it's not very well documented!)

Other than that LGTM, although Travis broke the open source world and so I need to either get OSS build credits from them or migrate to someone else before I can merge this.

Reply:
Thanks @strugee